syntax = "proto3";
package role;





/*
RoleSetUser请求
*/
message RoleSetUserRequest {
    /*
    用户(组)名
    */
    string user = 1 ;
    /*
    角色名列表
    */
    repeated string roles = 2 ;
}

/*
RoleSetUser返回
*/
message RoleSetUserResponse {
    /*
    成功设置的用户数量
    */
    int32 count = 1 ;
}

/*
RoleSetUserApi返回
*/
message RoleSetUserResponseWrapper {
    /*
    返回码
    */
    int32 code = 1 ;
    /*
    返回码解释
    */
    string codeExplain = 2 ;
    /*
    错误详情
    */
    string error = 3 ;
    /*
    返回数据
    */
    role.RoleSetUserResponse data = 4 ;
}

